### PR TITLE
add lead maintainer

### PR DESCRIPTION
**Pull request by diasdavid**
For https://github.com/ipfs/community/pull/301#issuecomment-385334835

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,10 +17,6 @@
     "coverage": "aegir coverage",
     "coverage-publish": "aegir coverage publish"
   },
-  "pre-commit": [
```
Comment (vmx) [inline package.json]: This should not be removed, but changed to pre-push. I'd also like to see a better commit message :)

**Comment by olizilla** [comment]
@diasdavid @vmx one thought on this before I accept the mantle of responsibility... I feel this specific transference of power might work better if the modules that make up the dependencies of js-cid were considered as a collection (or fiefdom) that were handed to a single lead maintainer, rather than split out... the js-cid maintainer is going to have a better time of it if they can also directly manage multibase, multicode and multihashes.

It look like js-cid didn't make the [power transference list](https://github.com/ipfs/pm/issues/600#issuecomment-385674334), was that deliberate?

In general I'm all for this distribution of responsibility, and if neither of you feel strongly about it, then I accept this responsibility gladly!

**Comment by diasdavid** [comment]
@olizilla, @vmx was still creating the PRs for the IPLD modules as requested, now it is up to date.

I don't think that because js-cid uses multibase, multicodec and multihashes, that it means that only one maintainer has to cover it all. That would mean that in order to maintain js-ipfs, you have to maintain the whole project (which i've been doing but now opening it for others to help). It actually encourages healthy updates from the multiple maintainers to make sure that the assumptions and goals are still the same.

File: package.json (hunk 1 above)
Comment (olizilla) [reply package.json]: @diasdavid why _was_ the `pre-commit` check removed?

@vmx is `pre-push` the new `pre-commit`? I don't think the [`pre-commit`](https://github.com/observing/pre-commit/blob/master/README.md) module we use supports `pre-push`.

Comment (vmx) [reply package.json]: @olizilla `pre-push` is indeed the new `pre-commit`. You don't need to add the package, it's part of AEgir.

**Comment by diasdavid** [comment]
image


@olizilla I see your thumbs up, I'll take that as a ok to move forward :)

**Pull request merged**